### ISSUE TITLE
Cannot draw the AMR plot with AMRPlot() function

### ISSUE THREAD

**Issue by ZihanChen1995** [bjascob/amrlib#42]
Hi Brad, thank you for this great library. However, I met a problem when I tried to draw the plot of the AMR graph with `AMRPlot()` function. May I ask how can I solve this problem?

---
I installed the package following the instruction page, and tested the sample code in Jupyter Notebook (Google Colab). It worked the job perfectly. 
```
stog = amrlib.load_stog_model(model_dir="/my_local_path_to_model/model_parse_xfm_bart_base-v0_1_0")
graphs = stog.parse_sents(['This is a test of the system and this is a second sentence.'])
print(graphs[0])

>>># ::snt This is a test of the system and this is a second sentence.
(a / and
      :op1 (t / test-01
            :ARG0 (t2 / this)
            :ARG1 (s / system))
      :op2 (s2 / sentence
            :ord (o / ordinal-entity
                  :value 2)
            :domain (t3 / this)))
```

Then I tried to draw the plot with `AMRPlot()`. **However, the graph did not show up, and there were no error messages**.
(I checked the sample in the [Docs](https://amrlib.readthedocs.io/en/latest/api_misc/#amr-plotting), but seems the `test.txt` has been removed. And I have installed `graphviz` and it was successfully imported.)

```
plot = AMRPlot()
plot.build_from_graph(graphs[0], debug=False)
plot.view()
```

**Would you mind giving me some hints on this issue? Thank you!**

**Comment by bjascob** [comment]
I would guess it has something to do with your graphviz / default viewer setup.  Either the default viewer isn't working or maybe it's closing immediately after opening (ie.. try `while 1: pass` after that call). 

When you do `plot.view()` it should create `/tmp/amr_graph.pdf` (on linux, Windows has a different temp directory).  You can look in the `/tmp` directory and view the pdf file manually.  You can also call `plot.render()` instead of `plot.view()` which will only create the file and will return the filename and path of the pdf file.

You can also change the plot format from pdf (the default) to something like .png.  To do this do `plot = AMRPlot(format='png')`.  GraphViz supports a number of formats.  See those docs but bmp, jpg,... probably work too.

**Comment by ZihanChen1995** [comment]
Thank you for your reply, Brad. I have tried your advice, and here are some results.

1. I double-checked the installation of `graphviz`. As you mentioned, we need to be aware of the non-python version. I make sure both have been installed, but this does not help.
2. **Finding the graph in `/tmp/` folder works.** Actaully, I have found both `amr_graph.gv.pdf` and `amr_graph.gv` there. And the pdf one is the one that we need. 
3.  Changing the format does not help. I have tried `png` and `jpg`, but none of them could work.

In summary, at this moment, 1. assigning each graph a corresponding name with `render_fn`, 2. using `plot.render()`, and 3. finding the pdf file. is the best solution. Thank you so much, Brad!
